mmaction/models/backbones/resnet3d.py

```diff
@@ -455,7 +624,7 @@ def __init__(self,
             temporal_stride = temporal_strides[i]
             dilation = dilations[i]
             planes = self.base_channels * 2**i
-            res_layer = self.make_res_layer(
+            res_layer = make_res_layer(
```

Comment:
This changes may influence https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/backbones/resnet3d_slowfast.py, since it inherits from `ResNet3d` and use `make_res_layer` as a inner function.

Reply:
Is it good to set `make_res_layer` as a static method? so we can call it with `ResNet3d.make_res_layer` in `class ResNet3dLayer`. Or directly make `ResNet3dLayer` inherit `ResNet3d` ?

Reply:
Thx, I will import `make_res_layer` in the `slowfast.py`. My original motivation for this change is to reuse more code (since making ResNet3dLayer a child class of ResNet3d seems not proper).

Reply:
What about the current resolution: make those funcs static & set ResNet3dLayer.func = ResNet3d.func? I'm not sure whether this style is accepted or not.

Reply:
It is fine to me. What do u think? @innerlee

Reply:
static method is ok